src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/OidcEntityType.php

```diff
@@ -0,0 +1,458 @@
+                                'data-help' => 'entity.edit.information.logoUrl',
+                                'data-parsley-urlstrict' => null,
+                                'data-parsley-trigger' => 'blur',
+                            ],
+                        ]
+                    )
+                    ->add(
+                        'nameNl',
+                        TextType::class,
+                        [
+                            'required' => false,
+                            'attr' => ['data-help' => 'entity.edit.information.nameNl'],
+                        ]
+                    )
+                    ->add(
+                        'descriptionNl',
+                        TextareaType::class,
+                        [
+                            'required' => false,
+                            'attr' => [
+                                'data-help' => 'entity.edit.information.descriptionNl',
+                                'rows' => 10,
+                            ],
+                        ]
+                    )
+                    ->add(
+                        'nameEn',
+                        TextType::class,
+                        [
+                            'required' => false,
+                            'attr' => ['data-help' => 'entity.edit.information.nameEn'],
+                        ]
+                    )
+                    ->add(
+                        'descriptionEn',
+                        TextareaType::class,
+                        [
+                            'required' => false,
+                            'attr' => [
+                                'data-help' => 'entity.edit.information.descriptionEn',
+                                'rows' => 10,
+                            ],
+                        ]
+                    )
+                    ->add(
+                        'applicationUrl',
+                        TextType::class,
+                        [
+                            'required' => false,
+                            'attr' => [
+                                'data-help' => 'entity.edit.information.applicationUrl',
+                                'data-parsley-urlstrict' => null,
+                                'data-parsley-trigger' => 'blur',
+                            ],
+                        ]
+                    )
+                    ->add(
+                        'eulaUrl',
+                        TextType::class,
+                        [
+                            'required' => false,
+                            'attr' => [
+                                'data-help' => 'entity.edit.information.eulaUrl',
+                                'data-parsley-urlstrict' => null,
+                                'data-parsley-trigger' => 'blur',
+                            ],
+                        ]
+                    )
+                    ->add(
+                        'enablePlayground',
```

Comment:
Maybe make this part of the redirectUri's component? Or place it close to it? As it will add/remove the playground option to the options. Or will you perform this task in the CommandHandler during form processing?

Reply:
I wanted to do this just before pushing data to manage